tests-e2e/fixture/fixture.go

```diff
@@ -248,7 +248,9 @@ func cleanUpOldHASApplicationAPIs(namespace string, k8sClient client.Client) err
 	for i := range applicationList.Items {
 		application := applicationList.Items[i]
 		if err := k8sClient.Delete(context.Background(), &application); err != nil {
-			return err
+			if !apierr.IsNotFound(err) {
```

Comment:
This (and below) should avoid race condition where the Delete was already completed.